src/envoy/http/authn/authn_utils_test.cc

```diff
@@ -58,6 +58,27 @@ const std::string kSecIstioAuthUserInfoHeaderWithAudValueArray =
          "some-other-string-claims": "some-claims-kept"
        }
      )";
+const std::string kSecIstioAuthUserInfoHeaderWithNestedClaims =
+    R"(
+       {
+         "iss": "issuer@foo.com",
+         "sub": "sub@foo.com",
+         "nested1": {
+           "aud1": "aud1a  aud1b",
+           "list1": ["list1a", "list1b"],
+           "other1": "str1",
+           "non-string-ignored": 111,
```

Comment:
Do we plan to support non-string claims going forward?

Reply:
maybe, the Istio API syntax will need update to support non-string match.